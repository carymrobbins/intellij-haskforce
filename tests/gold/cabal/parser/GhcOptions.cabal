library
  ghc-options: -Wall -fwarn-tabs -O2 -fdicts-cheap -funbox-strict-fields -fmax-simplifier-iterations=10 -w -threaded -rtsopts -with-rtsopts=-N