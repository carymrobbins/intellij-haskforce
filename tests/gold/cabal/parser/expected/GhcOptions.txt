Cabal File
  KEY_OR_CONFIG
    CONFIG
      LIBRARY
        PsiElement(LIBRARY)('library')
        LIBRARY_KEYS
          BUILD_INFORMATION
            GHC_OPTIONS
              PsiElement(CabalTokenType.ghc-options)('ghc-options')
              PsiElement(CabalTokenType.:)(':')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-Wall')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-fwarn-tabs')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-O2')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-fdicts-cheap')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-funbox-strict-fields')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-fmax-simplifier-iterations')
                PsiElement(CabalTokenType.=)('=')
                NUMBER
                  PsiElement(CabalTokenType.numberRegexp)('10')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-w')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-threaded')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-rtsopts')
              GHC_OPTION
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-with-rtsopts')
                PsiElement(CabalTokenType.=)('=')
                VARID
                  PsiElement(CabalTokenType.varidRegexp)('-N')